install-racket.sh

```diff
@@ -56,38 +65,52 @@ if [ -n "$TEST" ]; then
     exit 0
 fi
 
-# Older .travis.yml files don't set $RACKET_DIR (the Racket install
-# directory) explicitly and expect it to be /usr/racket.
-if [[ "$RACKET_DIR" = "" ]]; then
-    RACKET_DIR=/usr/racket
-fi
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    INSTALLER="./racket.dmg"
 
-INSTALLER="./racket-${MIN}${RACKET_VERSION}.sh"
+    echo "Downloading $URL to $INSTALLER:"
+    curl -L -o $INSTALLER $URL
 
-echo "Downloading $URL to $INSTALLER:"
-curl -L -o $INSTALLER $URL
+    echo "Mounting installer"
+    MOUNT=$(hdiutil attach "$INSTALLER" | grep Volumes | cut -f 3)
+    echo "Copying racket to ${RACKET_DIR}"
+    cp -a "${MOUNT}/Racket v${RACKET_VERSION}" "${RACKET_DIR}"
+    echo "Racket installed"
+else
 
-echo "Making $INSTALLER executable:"
-chmod u+rx "$INSTALLER"
+    # Older .travis.yml files don't set $RACKET_DIR (the Racket install
+    # directory) explicitly and expect it to be /usr/racket.
+    if [[ "$RACKET_DIR" = "" ]]; then
+        RACKET_DIR=/usr/racket
+    fi
 
-# Only use sudo if installing to /usr
-if [[ "$RACKET_DIR" = /usr* ]]; then
-    RUN_INSTALLER="sudo ${INSTALLER}"
-else
-    RUN_INSTALLER="${INSTALLER}"
-fi
+    INSTALLER="./racket-${MIN}${RACKET_VERSION}.sh"
+
+    echo "Downloading $URL to $INSTALLER:"
+    curl -L -o $INSTALLER $URL
+
+    echo "Making $INSTALLER executable:"
+    chmod u+rx "$INSTALLER"
 
-echo "Running $RUN_INSTALLER to install Racket:"
+    # Only use sudo if installing to /usr
+    if [[ "$RACKET_DIR" = /usr* ]]; then
+        RUN_INSTALLER="sudo ${INSTALLER}"
+    else
+        RUN_INSTALLER="${INSTALLER}"
+    fi
 
-$RUN_INSTALLER <<EOF
-no
-"$RACKET_DIR"
+    echo "Running $RUN_INSTALLER to install Racket:"
+
+    $RUN_INSTALLER <<EOF
+    no
+    "$RACKET_DIR"
 
 EOF
 
-if [[ "$RACKET_MINIMAL" = "1" ]]; then
-    echo "Minimal Racket: Installing packages for raco test..."
-    ${RACKET_DIR}/bin/raco pkg install --auto --scope installation rackunit-lib compiler-lib
+    if [[ "$RACKET_MINIMAL" = "1" ]]; then
```

Comment:
You have this nested under the non-Darwin else branch. But, should this be moved out to be done for Darwin, too?

I ask because it looks like you are including the minimal-racket scenario, up above. So, wouldn't you want this for minimal-Racket on Darwin, too?  (Apologies if I'm misunderstanding.)

Reply:
I'm sorry to be slow to follow up and review this. Partly I've been super busy. But also, I feel like adding the macOS "dimension" means the conditionals and permutations is getting to be a slightly heavy cognitive load.

I don't mean to sound like I'm complaining -- I really appreciate you taking the time to do and submit this PR. Thanks for that!!